tests/bluetooth/controller/common/src/helper_pdu.c

```diff
@@ -33,8 +33,47 @@
 
 
 #include "helper_pdu.h"
+#include "helper_features.h"
 
 
+void helper_pdu_encode_feature_req(struct pdu_data *pdu, void *param)
+{
+	pdu->ll_id = PDU_DATA_LLID_CTRL;
+	pdu->len = offsetof(struct pdu_data_llctrl, feature_req) +
+		sizeof(struct pdu_data_llctrl_feature_req);
+	pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_FEATURE_REQ;
+	for (int counter = 0; counter < 8; counter++) {
+		u8_t expected_value =
+			((u64_t) EXPECTED_FEATURE >> 8*counter) & 0xFF;
```

Comment:
Not liking this approach as we cannot control FeatureSet_S and FeatureSet_M to verify FeatureSet_USED logic. Same in all the other places where EXPECTED_FEATURE is used.

Reply:
I'll pass it in as a parameter, then we can control it from the unittests